lib/src/model/result.dart

```diff
@@ -16,6 +16,7 @@ class Result<Value, Failure> {
         success: (value) => value!,
         failure: (failure) {
           if (failure is Exception) {
+            // ignore: only_throw_errors
             throw failure;
```

Comment:
@pq this seems like a false positive. If I try to cast to Exception I get an unnecessary cast lint

Reply:
I think you're right.  Would you mind opening an issue on the linter?

https://github.com/dart-lang/linter/issues/new/choose

Reply:
Thanks!